mucommander-core/src/main/java/com/mucommander/search/SearchDialog.java

```diff
@@ -348,6 +419,11 @@ private String getSearchQuery() {
             if (lastTextRegex)
                 properties.add(new Pair<>(SearchBuilder.TEXT_MATCH_REGEX, Boolean.TRUE.toString()));
         }
+        if (lastSize != null) {
+            properties.add(new Pair<>(SearchBuilder.SEARCH_SIZE_REL, lastSizeRel.name()));
+            properties.add(new Pair<>(SearchBuilder.SEARCH_SIZE, lastSize.toString()));
+            properties.add(new Pair<>(SearchBuilder.SEARCH_SIZE_UNIT, lastSizeUnit.toString() ));
+        }
```

Comment:
the problem I see with this approach is that we'll be using a single 'size' condition per-search
I can see us adding the possibility to search for files where, e.g., size > 10MB and size < 20MB and then we'll need to replace this representation. what do you think about encoding this in a way that support multiple size conditions?

Reply:
the easiest change would be to have SEARCH_SIZE_REL2 SIZE_2... for a second condition.

Reply:
yeah, I guess we would have three options here:
1. search files with specific size (less common I guess)
2. search files with size lower than (or higher than) a certain value
3. search files with size that is between some min-value and max-value

so we won't  end up with more than two relations but naming the fields that way, with an index, will not be that great in the location field and the code for parsing it won't be that simple either

how about something like size=<:1:mb that indicates that we search for size that is lower than 1mb?
that way when handling the parsed properties for the search, we'll search for all occurrences of size properties and parse them the same way